README.md

```diff
@@ -1 +1,37 @@
-# internship-bootcamp-workshops
\ No newline at end of file
+# Workshop: Development and Coding Standards
+
+## Installation and execution the model code
+
+To manage python requirements we're using [Pipenv](https://github.com/pypa/pipenv#installation). On Fedora you should be
+good with running `sudo dnf install pipenv`.
+
+TODO: check if this works and if the interns have rhel CSB?!
```

Comment:
@fridex @vpavlin do you know what the interns run? CSB? If so, can any of you add instructions on how to install pipenv on CSB

Reply:
IIRC, it was CSB last year. Not sure about this year.